torch/_refs/__init__.py

```diff
@@ -3939,6 +3950,12 @@ def unbind(t: TensorLikeType, dim: int = 0) -> TensorSequenceType:
         )
 
 
+@register_decomposition(aten.unbind_copy)
+@out_wrapper()
+def unbind_copy(x: TensorLikeType, dim: int = 0):
+    return x.clone(memory_format=torch.contiguous_format).unbind(dim)
```

Comment:
I'm pretty sure this is not correct. `unbind_copy` will return `n` fresh storages. This one returns `n` view into one storage.